chunjun-core/src/main/java/com/dtstack/chunjun/util/ExternalDataUtil.java

```diff
@@ -0,0 +1,350 @@
+package com.dtstack.chunjun.util;
```

Comment:
The new file needs the apache license.

Reply:
@FlechazoW should i add the apache license by myself and commit another commit?

Reply:
yes, you should add the license. Don't worry about the commit, we will squash them, just commit.